### DISCUSSION TITLE
[async] How can I load relationship/s to an already loaded object?

### DISCUSSION THREAD

**Discussion by alexted** [sqlalchemy/sqlalchemy#8719] [Usage Questions] [answered]
Good day! Please tell me, how can I load relationship/s to an already loaded object in async mode?

**Comment by zzzeek** [comment, marked as answer]
ideally you would have loaded them already using [eager loading](https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html).  However, you can refresh those relationships using [populate_existing](https://docs.sqlalchemy.org/en/14/orm/queryguide.html#orm-queryguide-populate-existing) as well as the [refresh()](https://docs.sqlalchemy.org/en/14/orm/extensions/asyncio.html#sqlalchemy.ext.asyncio.AsyncSession.refresh) method.

**Reply by alexted** [reply]
thank you
